docs/reference-docs/cli.md

```diff
@@ -1,61 +1,40 @@
-# Command line interface commands
+# Command Line Interface Commands
 
 Top level options:
 
---install-completion [bash|zsh|fish|powershell|pwsh]
+`--install-completion [bash|zsh|fish|powershell|pwsh]`
 
 Install completion for the specified shell. [default: None]
 
---show-completion [bash|zsh|fish|powershell|pwsh]
+`--show-completion [bash|zsh|fish|powershell|pwsh]`
 
 Show completion for the specified shell, to copy it or customize the
 installation. [default: None]
 
 ## db
 
-Interact with the application database.
-
-### downgrade
-
-The `main.py db downgrade` command will downgrade the database to the previous
-revision or to the optionally specified revision.
-
-Arguments
-
-revision - Rev id to upgrade to [default: -1]
-
-### heads
-
-The `main.py db heads` command shows the Alembic database heads.
-
-### history
-
-The `main.py db history` command lists Alembic revision history in
-chronological order.
-
-Options
-
---verbose - Verbose output
---current - Indicate current revision
-
-### init
-
-The `main.py db init` command initialises an empty migrations environment.
-This command will throw an exception when it detects conflicting files and
-directories.
-
-### merge
-
-The `main.py db merge` command merges database revisions.
-
-Arguments
-
-revisions - Add the revision you would like to merge to this command
-[default: None]
-
-Options
-
---message TEXT - The revision message
+Interact with the application database. By default, does nothing, specify `main.py db --help` for more information.
+
+::: orchestrator.cli.database
+    options:
+      show_signature: false
+      show_root_heading: false
+      docstring_style: google
+      show_docstring_parameters: false
+      show_docstring_returns: false
+      show_root_toc_entry: false
+      show_symbol_type_toc: false
+      show_symbol_type_heading: false
+      members:
+        - downgrade
```

Comment:
I think this would be a more logical order for the commands
```suggestion
        - init
        - revision
        - upgrade
        - downgrade
        - heads
        - history
        - merge
        - migrate_workflows
```